data/json/mapgen/cemetery_4square.json

```diff
@@ -413,7 +413,8 @@
         { "item": "church", "x": [ 14, 18 ], "y": [ 5, 14 ], "chance": 65 },
         { "item": "church", "x": [ 11, 12 ], "y": [ 5, 6 ], "chance": 65 },
         { "item": "jackets", "x": [ 12 ], "y": [ 14 ], "chance": 50 }
-      ]
+      ],
+      "place_npcs": [ { "chance": 7, "class": "SEER_Brigitte_LaCroix", "x": 18, "y": 6 } ]
```

Comment:
FYI, `chance` is not a supported field in `place_npcs` (although perhaps it should be), and so this NPC will always spawn here.

Reply:
How else should it be done?

Reply:
I believe the only way to get a non-certain probability of NPC spawning right now is to use nested mapgen.  However, I think adding support for `chance` is probably the better solution in the long term.